content/master/concepts/patch-and-transform.md

```diff
@@ -0,0 +1,1648 @@
+
+Use the {{<hover label="math" line="6">}}math{{</hover>}} transform to multiply
+an input or apply a minimum or maximum value. 
+
+{{<hint "important">}}
+A {{<hover label="math" line="6">}}math{{</hover>}} transform only supports
+integer inputs. 
+{{< /hint >}}
+
+```yaml {label="math",copy-lines="1-7"}
+patches:
+  - type: FromCompositeFieldPath
+    fromFieldPath: spec.numberField
+    toFieldPath: metadata.annotations["mathAnnotation"]
+    transforms:
+      - type: math
+        math:
+          ...
+```
+
+<!-- vale Google.Headings = NO -->
+#### clampMin
+<!-- vale Google.Headings = YES -->
+
+The {{<hover label="clampMin" line="8">}}type: clampMin{{</hover>}} uses a defined 
+minimum value if an input is larger than the 
+{{<hover label="clampMin" line="8">}}type: clampMin{{</hover>}} value.
+
+For example, this 
+{{<hover label="clampMin" line="8">}}type: clampMin{{</hover>}} requires an
+input to be greater than 
+{{<hover label="clampMin" line="9">}}20{{</hover>}}.
+
+If an input is lower than 
+{{<hover label="clampMin" line="9">}}20{{</hover>}}, Crossplane uses the 
+{{<hover label="clampMin" line="9">}}clampMin{{</hover>}} value for the 
+{{<hover label="clampMin" line="4">}}toFieldPath{{</hover>}}.
+
+```yaml {label="clampMin"}
+patches:
+  - type: FromCompositeFieldPath
+    fromFieldPath: spec.numberField
+    toFieldPath: metadata.annotations["mathAnnotation"]
+    transforms:
+      - type: math
+        math:
+          type: clampMin
+          clampMin: 20
+```
+
+<!-- vale Google.Headings = NO -->
+#### clampMax
+<!-- vale Google.Headings = YES -->
+
+The {{<hover label="clampMax" line="8">}}type: clampMax{{</hover>}} uses a defined 
+minimum value if an input is larger than the 
+{{<hover label="clampMax" line="8">}}type: clampMax{{</hover>}} value.
+
+For example, this 
+{{<hover label="clampMax" line="8">}}type: clampMax{{</hover>}} requires an
+input to be less than 
+{{<hover label="clampMax" line="9">}}5{{</hover>}}.
+
+If an input is higher than 
+{{<hover label="clampMax" line="9">}}5{{</hover>}}, Crossplane uses the 
+{{<hover label="clampMax" line="9">}}clampMax{{</hover>}} value for the 
+{{<hover label="clampMax" line="4">}}toFieldPath{{</hover>}}.
+
+```yaml {label="clampMax"}
+patches:
+  - type: FromCompositeFieldPath
+    fromFieldPath: spec.numberField
+    toFieldPath: metadata.annotations["mathAnnotation"]
+    transforms:
+      - type: math
+        math:
+          type: clampMax
+          clampMax: 5
+```
+
+<!-- vale Google.Headings = NO -->
+#### Multiply
+<!-- vale Google.Headings = YES -->
+
+The {{<hover label="multiply" line="8">}}type: multiply{{</hover>}} multiplies
+the input by the {{<hover label="multiply" line="9">}}multiply{{</hover>}} 
+value.
+
+For example, this 
+{{<hover label="multiply" line="8">}}type: multiply{{</hover>}} multiplies the
+value from the {{<hover label="multiply" line="3">}}fromFieldPath{{</hover>}}
+value by {{<hover label="multiply" line="9">}}2{{</hover>}}
```

Comment:
Does multiply support floats? If so, we should call out that the lack of a "Divide" function is covered by multiplying decimals.

Reply:
Good point, I didn't think about that. It only supports integers. Note added.